pupil-spa/gulpfile.js

```diff
@@ -10,3 +10,11 @@ gulp.task('setRuntimeConfigURL', () => {
       .pipe(gulp.dest('./src/app/services/config/'))
   }
 })
+
+gulp.task('setSecretEnvVars', () => {
+  require('dotenv').config()
```

Comment:
i would require this at the top of the file, as per standard usage.  that way its obvious to all future contributors that it's there.

Reply:
@GuyHarwood done.